src/vm/checker/typecheck/tests/contracts.rs

```diff
@@ -187,12 +123,14 @@ fn test_names_tokens_contracts_2() {
     }
     result.unwrap();
 
-    let result = type_check(&"names", &mut names_contract, &mut db, true);
-    if let Err(ref e) = result { 
-        println!("{}", e);
-    } else {
-        panic!();
-    }
+    let err = type_check(&"names", &mut names_contract, &mut db, true).expect_err("Expected type error.");
+    assert!(match &err.err {
+            &CheckErrors::TypeError(ref expected_type, ref actual_type) => {
+                eprintln!("Received TypeError on: {} {}", expected_type, actual_type);
+                format!("{} {}", expected_type, actual_type) == "int bool"
```

Comment:
I guess I'm having a hard time seeing why this is broken -- doesn't the `broken-public` function evaluate to a `ResponseType<Int, Int>`?

Reply:
Ah -- it's this call:

```
 (contract-call! tokens token-transfer
                 burn-address 'true)
```

It's passing a bad parameter to `token-transfer`

Reply:
D'oh! missed that